shared/wallets/onboarding/disclaimer.tsx

```diff
@@ -214,88 +187,74 @@ class Disclaimer extends React.Component<DisclaimerProps, DisclaimerState> {
-      our servers. cool!
-    </Kb.Text>
-
-    <Kb.Text type="Body" style={Styles.collapseStyles([styles.bodyText, styles.bodyBullet])}>
-      • we can help you send and receive crypto just by knowing usernames. You can say goodbye to ugly
-      "addresses" you have to pass around insecurely.
+      1. WHAT IS STELLAR? Stellar is an open financial network. Anyone can use Stellar to send payments, issue
+      tokens, create smart contracts, store value, and other activities on the Stellar network. Stellar is
+      powered by free, open-source software and is hosted by independent computers across the globe. No one
+      entity can control the Stellar network.
     </Kb.Text>
-
     <Kb.Text type="Body" style={styles.bodyText}>
-      And we believe Stellar is in{' '}
-      <Kb.Text style={styles.bodyText} type="BodyItalic">
-        a unique position
-      </Kb.Text>{' '}
-      to solve payments because:
-    </Kb.Text>
-
-    <Kb.Text type="Body" style={Styles.collapseStyles([styles.bodyText, styles.bodyBullet])}>
-      • it's ultra fast and ultra cheap
-    </Kb.Text>
-
-    <Kb.Text type="Body" style={Styles.collapseStyles([styles.bodyText, styles.bodyBullet])}>
-      • it natively understands currencies and tokens
-    </Kb.Text>
-
-    <Kb.Text type="Body" style={Styles.collapseStyles([styles.bodyText, styles.bodyBullet])}>
-      • the network itself has a decentralized exchange built into it
-    </Kb.Text>
-
-    <Kb.Text type="Body" style={Styles.collapseStyles([styles.bodyText, styles.bodyBullet])}>
-      • it doesn't burn more electricity than small nations
-    </Kb.Text>
-
-    <Kb.Text type="Body" style={Styles.collapseStyles([styles.bodyText, styles.bodyBullet])}>
-      But there are a few things you must agree to understand before using Stellar on Keybase:
+      2. WHAT IS A LUMEN? Lumens (also referred to as "XLM") are cryptocurrency that are the native asset of
+      the Stellar network. Lumens are required to use the Stellar network: they’re needed for transaction
+      fees, account maintenance fees, and more. Read more about how you can use Lumens here:{' '}
+      <Kb.Text
+        type="BodyPrimaryLink"
+        onClick={() => openURL('https://www.stellar.org/lumens/')}
+        style={styles.bodyText}
+      >
+        https://www.stellar.org/lumens/
+      </Kb.Text>
+      . Note that Lumens can always be used for the Stellar network, but their fiat value could be highly
+      volatile.
     </Kb.Text>
-
     <Kb.Text type="Body" style={styles.bodyText}>
-      1. IT'S BRAND NEW AND YOU ARE AMONG ITS FIRST TESTERS. Seriously, don't race off and buy more
-      cryptocurrency than you're willing to lose. And don't manage tokens in Keybase that you're not willing
-      to lose. We could have an exploitable bug in an early release. You're using this app at your own risk.{' '}
+      3. KEYBASE'S STELLAR WALLET IS BRAND NEW, AND YOU ARE AMONG ITS FIRST TESTERS. Seriously, don't race off
```

Comment:
this isn't really true anymore, is it?

Reply:
Fair! @malgorithms, want to keep or remove this point?

Reply:
sure, we can remove.